skymp5-front/src/constructor.js

```diff
@@ -27,6 +27,7 @@ const Constructor = props => {
 
   useEffect(() => {
     if (props.dynamicSize) {
```

Comment:
Let's rewrite this part, please use switch for `props.elem.type`. we could extract long if conditions `(content_mainRef && content_mainRef.current && content_mainRef.current.clientHeight && content_mainRef.current.clientWidth)` to a separate constant.